packages/ui/src/working-groups/model/getAverageStake.ts

```diff
@@ -0,0 +1,9 @@
+import BN from 'bn.js'
+
+import { WorkerFieldsFragment } from '@/working-groups/queries'
+
+export const getAverageStake = (workers: Pick<WorkerFieldsFragment, 'stake'>[]): BN => {
+  const totalStake = workers.reduce((a, b) => a + b.stake, 0)
```

Comment:
Similarly to the other PR - you can use `BN` here.

Reply:
✅